### ISSUE TITLE
Table panel: scrolling dashboard should take precedence

### ISSUE THREAD

**Issue by ssbarnea** [grafana/grafana#21324]
**What happened**:
The removal of variable height table support and scrolling introduces a serious UX issue for people using Table Panel on big dashboards that require scrolling.

**What you expected to happen**:

Scrolling the dashboard page itself should always take priority, regardless if the mouse cursor is above a Table Panel.

**How to reproduce it (as minimally and precisely as possible)**:

Take a look at http://dashboard-ci.tripleo.org/d/YRJtmtNWk/cockpit?orgId=1 and try to scroll up and down on it. You will soon realize that there are only two very narrow colums that can be used to perform a full scroll without being affected by the table-scrolling. 

The sidebar breaks the scroll, same any table, this means that about 2-3% of the screen read estate can be used to perform the scrolling.

Ideally scrolling inside the tables should be triggered using a modifier key or the UI should be changed in order to avoid breaking webpage scrolling.

**Anything else we need to know?**:
It does also reproduce with the latest release.

# References
* https://community.grafana.com/t/table-panel-auto-height-column-names/10483/3
* https://github.com/grafana/grafana/pull/18318 which caused this bug

**Comment by dprokop** [comment]
@ssbarnea thanks for reporting this! I see your point but no quick solution comes to my mind at the moment. Haven't seen any reports so far about this being an issue, so let me put this into the backlog an we will prioritise the work on it.

**Comment by ssbarnea** [comment]
@dprokop Thanks for the quick reply. It would be very useful to consult with others and at least to indicate some possible ways you want to approach it, even if implementation could be delayed a lot. I guess most users are not "abusing" the table-panels like us.

I consulted my team regarding a potential reduction of default 100 rows to something that would fit the panel heigh but this causes other problems:
* using pagination is problematic when you try to investigate when a build started to fail. If we set it to ~10 rows to match the average height we currently have it would be a huge PITA to look-back.
* number of rows to fit depends on browser or zoom-level
